src/themes/components/input.ts

```diff
@@ -16,23 +13,19 @@ const baseStyle: PartsStyleObject<typeof parts> = {
     appearance: 'none',
     transitionProperty: 'common',
     transitionDuration: 'normal',
+    px: 'md',
+    py: 'xs',
   },
 };
 
 const size: Record<string, SystemStyleObject> = {
   md: {
-    fontSize: 'lg',
-    px: 'md',
-    py: 'xs',
-    h: sizeDefinitions.md,
-    borderRadius: 'md',
```

Comment:
We don't have any `md` borderRadius token. It should be `sm` for all the variations. 😱